core/include/lifo.h

```diff
@@ -23,32 +24,47 @@
 #ifndef LIFO_H
 #define LIFO_H
 
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 /**
- * @brief   Check if the given lifo is empty.
+ * @brief       Check if the given lifo is empty.
+ * @deprecated  will be removed with release 2021.07
```

Comment:
I would prefer not documenting *every* function as deprecated, last time we did this, the [deprecated list](https://doc.riot-os.org/deprecated.html) became an chaotic mess.

Reply:
I am ok with either,
but i just had a look into the deprecated list and it seems like its common practice to have multiple deprecations per file.

Reply:
changed